LeetCode Problems/2601 to 2700/2641. Cousins in Binary Tree II/2641. Cousins in Binary Tree II.cpp

```diff
@@ -0,0 +1,53 @@
+class Solution {
+public:
+    vector<int> sums;
+    deque<TreeNode*> dq;
+
+    void bfs(TreeNode* root){
+        dq.push_back(root);
+        int lev = 0;
+
+        while(!dq.empty()){
+            int n = dq.size();
+            while(n--){
+                TreeNode* node = dq.front();
+                int sibSum = 0;
+                if(node->left){
+                    sibSum += node->left->val;
+                }
+                if(node -> right){
+                    sibSum += node->right->val;
+                }
+                if(node -> left){
+                    node->left->val = sums[lev+1] - sibSum;
+                    dq.push_back(node->left);
+                }
+                if(node->right){
+                    node->right->val = sums[lev+1] - sibSum;
+                    dq.push_back(node->right);
+                }
+                dq.pop_front();
+            }
+            lev++;
+        }
+    }
+
+    void dfs(TreeNode* root, int lev){
+        if(root == nullptr) return;
+        if(sums.size() > lev){
+            sums[lev] += root->val;
+        }
+        else{
+            sums.push_back(root->val);
+        }
+        dfs(root->left, lev+1);
+        dfs(root->right, lev+1);
+    }
+
+    TreeNode* replaceValueInTree(TreeNode* root) {
+        dfs(root, 0);
+        bfs(root);
+        root->val = 0;
+        return root;
+    }
+};
```

Comment:
Hi @simran0963 
Thanks for your awesome work.

But your code does not follow our **```clang format style```**.
So I think you will solve this ```format``` issue to make this **```PR```** suitable for  **```Merge```** .